android/build.gradle

```diff
@@ -31,10 +31,10 @@ def safeExtGet(prop, fallback) {
 }
 
 android {
-    compileSdkVersion safeExtGet('Hmssdk_compileSdkVersion', 30)
+    compileSdkVersion safeExtGet('Hmssdk_compileSdkVersion', 31)
```

Comment:
@stanwolverine why is updating this compile SDK version required?
Clients who have not updated to 31 won't be able to compile the app.